src/components/CurrentUser/CurrentUser.tsx

```diff
@@ -1,12 +1,59 @@
 import React from 'react';
 import './CurrentUser.scss';
+import { getUsers } from '../../api';
+import { User } from '../types/User';
 
-export const CurrentUser: React.FC = () => (
-  <div className="CurrentUser">
-    <h2 className="CurrentUser__title"><span>Selected user: 2</span></h2>
+type Props = {
+  userId: number,
+};
 
-    <h3 className="CurrentUser__name">Ervin Howell</h3>
-    <p className="CurrentUser__email">Shanna@melissa.tv</p>
-    <p className="CurrentUser__phone">010-692-6593 x09125</p>
-  </div>
-);
+type State = {
+  user: User | null,
+};
+
+export class CurrentUser extends React.Component<Props, State> {
+  state:State = {
+    user: null,
+  };
+
+  componentDidMount() {
+    getUsers(this.props.userId)
+      .then(response => {
+        this.setState({ user: response });
+      });
```

Comment:
Move this code to some method so you won't repeat yourself in componentDidUpdate